introduction.md

```diff
@@ -1,2 +1,6 @@
 Hi, this is an introduction!
 My favorite Code School path is JavaScript
+My deadly skills are:
+*Git
+*JavaScript
+*Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?